src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -194,18 +196,20 @@ internal static LanguageVersion RequiredVersion(this MessageID feature)
             // Checks are in the LanguageParser unless otherwise noted.
             switch (feature)
             {
+                //PROTOTYPE(readonlyRefs): these are all 7.2 features we need to update this.
+                // C# 7.1 features.
+                case MessageID.IDS_FeatureReadonlyReferences:
+                case MessageID.IDS_FeatureRefStructs:
+                case MessageID.IDS_FeatureReadonlyStructs:
+                    return LanguageVersion.CSharp7_1;
```

Comment:
Evidently the features are still marked as 7_1. 
I did not want to change that in this change since it is completely orthogonal. It used to be useful to have them as 7.1 for dogfooding reasons, but not any more. We should fix this soon.

---
In reply to: [118551720](https://github.com/dotnet/roslyn/pull/19713#discussion_r118551720) [](ancestors = 118551720)